roles/ceph-rgw/defaults/main.yml

```diff
@@ -22,6 +22,10 @@ cephx: true
 # allowing root to not require tty
 radosgw_user: root
 
+# Multi-site remote pull URL variables
+rgw_pullport: "{{ radosgw_civetweb_port }}"
```

Comment:
Sure, I was just keeping with the current standard: rgw_pull_host, but can change it if you like.